docs/developers/node-tutorial.mdx

```diff
@@ -1606,6 +1606,19 @@ Get data availability sampler stats:
 celestia rpc das SamplingStats
 ```
 
+Transfer balance of utia to another account. First, set your address as a variable:
+
+```bash
+export ADDRESS=celestia1c425ckmve2489atttx022qpc02gxspa29wmh0d
+celestia rpc state Transfer $ADDRESS 100000 8000 80000
```

Comment:
[optional] it's not immediately obvious what the `8000` and `80000` values are for and the help docs don't describe the params (created https://github.com/celestiaorg/celestia-node/issues/2493) so it may be nice to describe the params in the sentence above but not blocking.